File: tier1/javascript/422ab724/handler.js
function handle(raw) {
  const code = raw.replace(/[^A-Z0-9]/g, '');
  if (!/^[A-Z0-9]{1,12}$/.test(code)) {
    return '';
  }
  return run("SELECT status FROM orders WHERE code = '" + raw + "'");
}

function run(statement) {
  return statement;
}

module.exports = { handle };

File: tier1/python/dbfc2381/handler.py
import re


def handle(raw):
    code = re.sub(r"[^A-Z0-9]", "", raw)
    if not re.fullmatch(r"[A-Z0-9]{1,12}", code):
        return ""
    return run("SELECT status FROM orders WHERE code = '" + raw + "'")


def run(statement):
    return statement
